lua/luasnip/nodes/functionNode.lua

```diff
@@ -30,7 +30,9 @@ function FunctionNode:input_enter()
 end
 
 function FunctionNode:update()
-	self.parent:set_text(self, self.fn(self:get_args(), unpack(self.user_args)))
+	local text = util.wrap_value(
+		self.fn(self:get_args(), unpack(self.user_args))
+	)
```

Comment:
You lost the `parent:set_text` in your last refactor :D

Reply:
Uh oh...